### ISSUE TITLE
AccessibilityChecker.org should be tagged as limited free functionality, not free

### ISSUE THREAD

**Issue by shawna-slh** [w3c/wai-evaluation-tools-list#903]
From email to wai@w3.org:
> Thank you for developing such a useful list of accessibility tools. 

> Just wanted to flag that AccessibilityChecker.org has limited free functionality - you get one free PDF download by entering your email address, and it seems to allow only 2-3 free site scans before showing a paywall. 

> It is currently under the free category but I believe it should be 'limited free functionality' instead.
